py/torch_tensorrt/_compile.py

```diff
@@ -502,19 +502,24 @@ def save(
                 "Provided model is a torch.jit.ScriptModule but the output_format specified is exported_program. Please verify the output_format"
             )
         else:
+            if arg_inputs is not None:
+                raise ValueError(
+                    "Provided model is a torch.jit.ScriptModule, do not allow user to provide inputs or arg_inputs."
```

Comment:
This needs to be rephased to be more user facing

e.g "Provided model is a torch.jit.ScriptModule, arg_inputs is not necessary"

Also is this a real error or can it just be a warning?

Reply:
changed to warning.
add the warning is to let user know: 
if user provide input1 for torch_tensorrt.dynamo.compile(*****)
and then user call torch_tensorrt.save(****) with input2
since the torch_tensorrt model is compiled with input1, when save it is not going to use the input2